metadata.json

```diff
@@ -8,9 +8,9 @@
   "project_page": "https://github.com/puppetlabs/puppetlabs-postgresql",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":"4.x"},
-    {"name":"puppetlabs/apt","version_requirement":">=2.0.0 <3.0.0"},
-    {"name":"puppetlabs/concat","version_requirement":">= 1.1.0 <3.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.13.1 < 5.0.0"},
+    {"name":"puppetlabs/apt","version_requirement":">= 2.0.0 < 3.0.0"},
+    {"name":"puppetlabs/concat","version_requirement":">= 1.1.0 < 3.0.0"}
```

Comment:
I'm happily using it with concat4. but bumping this shouldn't be part of this PR.